prow/monitoring/grafana_deployment.yaml

```diff
@@ -40,6 +40,9 @@ spec:
         - mountPath: /etc/grafana
           name: grafana-config
           readOnly: false
+        - mountPath: /grafana-dashboard-definitions/0/prow
```

Comment:
Why does this path have a `prow` suffix?

Reply:
IIRC it's the name of the group of dashboards?